using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference.md

```diff
@@ -270,21 +289,27 @@ public void onResume() {
 
 {% tab title="iOS (ACP 2.x)" %}
 {% hint style="info"}
-You can pass the `launchOptions` that was received upon opening the application or `userInfo`, which contains the received push payload in `trackInfo`. If `trackInfo` is null or does not contain the necessary tracking identifiers, `broadlogId` (`_mId`) and `deliveryId` (`_dId`), a track request is **not** sent.
+You can pass the `launchOptions` that were received upon opening the application or `userInfo`, which contains the received push payload in `trackInfo`. If `trackInfo` is null or does not contain the necessary tracking identifiers, `broadlogId` (`_mId`) and `deliveryId` (`_dId`), a track request is **not** sent.
 {% endhint %}
 
 ### Swift
 
 **Syntax**
 
 ```swift
-func trackNotificationClick(trackInfo: [String: String])
+func trackNotificationClick(_ trackInfo: [String: String])
 ```
 
 **Example**
 
 ```swift
-ACPCampaignClassic.trackNotificationClick(trackInfo[String:String])
+func userNotificationCenter(_ center: UNUserNotificationCenter, didReceive response: UNNotificationResponse, withCompletionHandler completionHandler: @escaping () -> Void) {
+      guard let userInfo = response.notification.request.content.userInfo as? [String: String] else {
+          return
+      }
+      ACPCampaignClassic.trackNotificationClick(userInfo)
+      completionHandler()
+}
 ```
 
 ### Objective-C
```

Comment:
the format for the obj-c example below is a little off L330-331, can we fix that too?
```
[ACPCampaignClassic trackNotificationClick:response.notification.request.content.userInfo];
completionHandler();
```